environment.yml

```diff
@@ -0,0 +1,195 @@
+name: py_epi
```

Comment:
Thanks for the PR Sarada! ❤️

Did you intend to include the `environment.yml` file? Would you mind removing it, then I'm very happy to merge this! 😄

Reply:
Sorry, my bad.  Now, the .yml file is deleted.